tooling/v8-snapshot/test/utils/bundle.ts

```diff
@@ -12,7 +12,7 @@ export function readBundleResult (cacheDir: string) {
 
 export function readSnapshotResult (cacheDir: string) {
   const metaFile = path.join(cacheDir, 'snapshot-meta.json')
-  const meta = require(metaFile)
+  const meta = fs.readJSONSync(metaFile)
```

Comment:
Does the "no synchronize file operations" rule apply here for the v8 tooling?

https://github.com/cypress-io/cypress/blob/develop/CONTRIBUTING.md#enfile-or-emfile

Reply:
I believe we're only concerned with this at runtime (so tests and scripts don't worry about it):

https://github.com/cypress-io/cypress/blob/develop/.eslintrc.js#L83
https://github.com/cypress-io/cypress/blob/develop/.eslintrc.js#L41-L47